src/pkg/packaging/windows/package.targets

```diff
@@ -47,6 +47,10 @@
         <Output TaskParameter="GeneratedGui" PropertyName="HostFxrUpgradeCode" />
       </GenerateGuidFromName>
 
+      <GenerateGuidFromName Name="$(TargetingPackInstallerFile)">
+        <Output TaskParameter="GeneratedGui" PropertyName="TargetingPackUpgradeCode" />
```

Comment:
Gui?

Reply:
I might as well fix it in this PR. Nothing points to this being anything but a typo.